packages/fx-core/src/component/driver/deploy/azure/impl/azureDeployImpl.ts

```diff
@@ -139,10 +138,14 @@ export abstract class AzureDeployImpl extends BaseDeployImpl {
           await waitSeconds(DeployConstant.BACKOFF_TIME_S);
         } else if (res?.status === HttpStatusCode.OK || res?.status === HttpStatusCode.CREATED) {
           if (res.data?.status === DeployStatus.Failed) {
-            await logger.error(
-              `Deployment is failed with error message: ${JSON.stringify(res.data)}`
+            await this.logger.warning(
+              getDefaultString(
+                "error.deploy.DeployRemoteStartError",
+                location,
+                JSON.stringify(res.data)
+              )
             );
-            throw new DeployRemoteStartError(location, JSON.stringify(res.data), this.helpLink);
+            // throw new DeployRemoteStartError(location, JSON.stringify(res.data), this.helpLink);
```

Comment:
If we don't use this code, it is better to delete it instead of comment it out

Reply:
Removed